templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Delete Car</h1>
+
+  <p>Are you sure you want to delete the car: {{ car }}?</p>
+  <p><i>All drivers with this car will be also deleted!</i></p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete the car." class="btn btn-danger" />
+  </form>
```

Comment:
Add a `Cancel` button here. The `Cancel` button should lead to the previous page the user was on. This will improve the user experience by giving them an option to cancel the deletion of a car.